code/modules/surgery/organs/stomach/_stomach.dm

```diff
@@ -314,5 +314,15 @@
 	if(prob(emp_vulnerability/severity)) //Chance of permanent effects
 		organ_flags |= ORGAN_SYNTHETIC_EMP //Starts organ faliure - gonna need replacing soon.
 
+/obj/item/organ/stomach/lizard
+	name = "lizard stomach"
+	icon_state = "stomach"
+	desc = "Lizards have evolved highly efficient stomachs, made to get nutrients out of what they eat as fast as possible."
+	metabolism_efficiency = 0.07
+
+/obj/item/organ/stomach/lizard/handle_hunger(mob/living/carbon/human/human, delta_time, times_fired)
+	. = ..()
+	if(human.nutrition > NUTRITION_LEVEL_WELL_FED && human.nutrition < NUTRITION_LEVEL_FULL)
+		human.adjustBruteLoss(-0.5 * delta_time)
```

Comment:
you should probably slap a nutrition cost on this

Reply:
i dont want to currently do this